### DISCUSSION TITLE
Is there any possibility that a Korean language pack will be added?

### DISCUSSION THREAD

**Discussion by moongcci** [pglombardo/PasswordPusher#926] [Q&A] [answered]
hello.
I'm a Korean.
I currently build and use pwpush at my company. We're making good use of it.
Are there any plans to add Korean language packs?

Thank you.

**Comment by pglombardo** [comment]
Hi @moongcci - I'd be very happy to add Korean support but we rely on volunteer translators.  Would you be interested in translating or at least verifying a Korean translation?

Some information on our translation process:
https://pwpush.com/en/pages/translate

**Comment by pglombardo** [comment, marked as answer]
Update on this:  Korean language has been added!  https://pwpush.com/ko/p/new

Thanks for posting the request.
